src/runtime.js

```diff
@@ -93,6 +94,9 @@ export function installAmpdocServices(ampdoc) {
   // TODO(dvoytenko, #3742): Split into runtime and ampdoc services.
   installActionServiceForDoc(ampdoc);
   installStandardActionsForDoc(ampdoc);
+  if (isExperimentOn(ampdoc.win, 'amp-lightbox-viewer')) {
```

Comment:
done. ended up installing the extension in `adoptShared`'s `waitForBody`. Not sure if there is a better place to do it?